reference/global_refs.yaml

```diff
@@ -1,9 +1,14 @@
 openapi: 3.1.0
```

Comment:
I'm going to split Global Refs into its own ticket, since it doesn't deal with negative numbers and I have more questions.